### PR TITLE
Travel to work indicators

### LINKED ISSUE #369: Travel to Work indicator
Issue for travel to work indicator only (travel connectivity indicators done)

### PR DESCRIPTION
**Pull request by dede95**
Travel to work indicators, schemas and notebook. 

Closes #369

### CODE REVIEW

File: ds/data/processed/travel/travel_to_airport_time.lep.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 api_type: FETCH
 title: Travel time to airport
 subtitle: Minimum travel time to nearest airport from a LEP region for England (2011 & 2013) and Wales (2011 only)
+description: Minimum travel time to nearest airport from a LEP region for England (2011 & 2013) and Wales (2011 only). Note: Overlapping LEP regions are used therefore some calculations may be under-estimated or over-estimated.
```
Comment (mindrones) [inline ds/data/processed/travel/travel_to_airport_time.lep.yaml]: (1) We need to have the `description` value within single apices as the presence of `:` causes an error when running the script updating the data for the app

File: ds/data/processed/travel/travel_to_rail_time.lep.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 api_type: FETCH
 title: Travel time to rail station
 subtitle: Minimum travel time to nearest rail station from a LEP region for England (2011 & 2013) and Wales (2011 only)
+description: Minimum travel time to nearest rail station from a LEP region for England (2011 & 2013) and Wales (2011 only). Note: Overlapping LEP regions are used therefore some calculations may be under-estimated or over-estimated.
```
Comment (mindrones) [inline ds/data/processed/travel/travel_to_rail_time.lep.yaml]: See (1)

File: ds/data/processed/travel/travel_to_road_junctions_average_time.lep.yaml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 api_type: FETCH
 title: Average time to nearest five road junctions
 subtitle: Average minimum travel time to nearest five road junctions from a LEP region for England (2011 & 2013) and Wales (2011 only)
+description: Average minimum travel time to nearest five road junctions from a LEP region for England (2011 & 2013) and Wales (2011 only). Note: Overlapping LEP regions are used therefore some calculations may be under-estimated or over-estimated.
```
Comment (mindrones) [inline ds/data/processed/travel/travel_to_road_junctions_average_time.lep.yaml]: See (1)

File: ds/data/processed/travel/travel_to_work_times_average.lep.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+api_type: FETCH
+title: Average home to work travel time in minutes
+subtitle: Average home to work travel time where LEP region is home location (October to December).
+description: Average home to work travel time where LEP region is home location observed from October to December each year. Note: Overlapping LEP regions are used therefore some calculations may be under-estimated or over-estimated.
```
Comment (mindrones) [inline ds/data/processed/travel/travel_to_work_times_average.lep.yaml]: See (1)

File: ds/data/processed/travel/travel_to_work_times_average.nuts2.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+api_type: FETCH
+title: Average home to work travel time in minutes
+subtitle: Average home to work travel time where NUTS 2 region is home location (October to December)
+endpoint_url:
```
Comment (mindrones) [inline ds/data/processed/travel/travel_to_work_times_average.nuts2.yaml]: no `description` here?

**Review by mindrones — CHANGES_REQUESTED** [review]
We need to adjust some of the YAML files to avoid errors during the build, some change required.

File: ds/data/processed/travel/travel_to_work_times_average.nuts2.yaml (hunk 5 above)
Comment (dede95) [reply ds/data/processed/travel/travel_to_work_times_average.nuts2.yaml]: No, I put a description in the LEP files to explain the chance of over-/under-estimation.

Comment (mindrones) [reply ds/data/processed/travel/travel_to_work_times_average.nuts2.yaml]: Ah I see, what about duplicating it here so that it's visible in the app too?

**Review by mindrones — APPROVED** [review]
We discussed about renaming:
• `travel_mean_time_to_road_junctions.*` to `travel_time_to_road_junctions.*`
• `travel_mean_time_to_work.*` to `travel_time_to_work.*`
for consistency with other similar indicators, but that's not a showstopper.

Feel free to merge! 👍

**Comment by mindrones** [comment]
Thanks for the rename, much better! (Good to merge of course :)

**Pull request merged**